qutip/solve/floquet.py

```diff
@@ -1032,15 +1033,23 @@ def fmmesolve(H, rho0, tlist, c_ops=[], e_ops=[], spectra_cb=[], T=None,
     else:
         w_th = 0
 
-    # TODO: loop over input c_ops and spectra_cb, calculate one R for each set
-
-    # calculate the rate-matrices for the floquet-markov master equation
-    Delta, X, Gamma, Amat = floquet_master_equation_rates(
-        f_modes_0, f_energies, c_ops[0], H, T, args, spectra_cb[0],
-        w_th, kmax, f_modes_table_t)
+    # floquet-markov master equation tensor
+    R = None
+    # loop over input c_ops and spectra_cb, calculate one R for each set
+    for c_op, spectrum in zip(c_ops, spectra_cb):
```

Comment:
If there are no `spectra_cb` passed to `fmmesolve` it automatically adds a white-noise spectrum for each `c_op`. Should I remove this behavior and require that the number of `c_ops` and `spectra_cb` is equal?

Reply:
Keep the adding of the white noise spectra, we just need to check so that `zip` doesn't silently truncate `c_ops` and `spectra_cb` to the shorter of the two lists.